channels/hn.m3u

```diff
@@ -77,5 +91,7 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/vid
 https://emisoras.hn:8081/tvcopan/index.m3u8
 #EXTINF:-1 tvg-id="UNAHUTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/De5CSD3.png" group-title="",UNAH UTV (360p) [Not 24/7]
 https://live-utv.unah.edu.hn/web/salida.m3u8
+#EXTINF:-1 tvg-id="VidaTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="Religious",Vida TV
+http://184.173.181.2:1935/8070/8070/playlist.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/147909743-6744699c-72f7-4e0a-9ff2-6d263d424ffa.png)

Reply:
Low bandwidth, but seems to answer.

Reply:
![image](https://user-images.githubusercontent.com/5430116/147957792-6b5fdfcf-1446-4b40-a2b4-1e2f8e49c884.png)

Reply:
Still doesn't work for me
![image](https://user-images.githubusercontent.com/7253922/148055982-c549051c-154f-4038-8d36-ccd2193d0183.png)

Reply:
Most certainly not 24/7

Reply:
it's better to delete it.

Reply:
Works again. Please mark as Not 24/7 @fershady 
![image](https://user-images.githubusercontent.com/5430116/148105910-47687c3b-7474-4751-8304-b57319708ff8.png)